src/storybook/decorators.tsx

```diff
@@ -2,9 +2,22 @@ import React from 'react'
 // also exported from '@storybook/react' if you can deal with breaking changes in 6.1
 import { Frame } from 'components/common/Frame'
 import { StoryFnReactReturnType } from '@storybook/react/dist/client/preview/types'
+import { ApolloProvider } from '@apollo/client'
+import { ApolloClient, InMemoryCache } from '@apollo/client'
 
 export const CenteredAndFramed = (DecoratedStory: () => StoryFnReactReturnType): JSX.Element => (
   <div style={{ textAlign: 'center' }}>
     <Frame style={{ display: 'inline-block' }}>{DecoratedStory()}</Frame>
   </div>
 )
+
+const apolloClient = new ApolloClient({
+  uri: 'https://api.thegraph.com/subgraphs/name/gnosis/protocol',
+  cache: new InMemoryCache(),
```

Comment:
Depending on a lot of stuff, if we use Apollo outside of storybook in the actual app, we may need to setup cache invalidation, or subscriptions, or something

I suspect this will just keep resolving queries from cache after first fetch.

Reply:
Good point. Should we create an issue with all the things I mentioned in my description, plus this, so we keep track of them as we learn more about apollo?

This PR was not part of our sprint, so I didn't go too fine grained into details. This `InMemoryCache` works great for now :P